src/utils/metadata.py

```diff
@@ -101,13 +101,17 @@ def build_metadata_from_setuptools_dict(metadata):
         homepage_url = metadata["url"]
     else:
         homepage_url = project_urls["Homepage"]
+    if "author" in metadata:
```

Comment:
you don't want to do this test in the reverse way? ie: check for `author-email`, and have `author` as the default?

Reply:
We can if you prefer

Reply:
hmm, I guess it's neither here nor there.